arch-gnome.sh

```diff
@@ -174,7 +174,7 @@ display_drivers()
 		print_progress_text "Installing nVidia video drivers"
 		echo -e "If prompted to select provider(s), select default options"
 		echo ""
-		sudo pacman -S nvidia lib32-virtualgl lib32-nvidia-utils
+		sudo pacman -S nvidia lib32-virtualgl nvidia-utils mesa nvidia-settings
```

Comment:
Thanks for the suggestions. Went back and looked at this:
* The `lib32-nvidia-utils` is effectively not required except for 32-bit (but will be pulled in by e.g. `steam` as a dependency).
* `lib32-virtualgl` is actually redundant
* `nvidia-utils` is a direct dependency of the `nvidia` package, and therefore doesn't need to be explicitly installed
* I hadn't included the `mesa` package here, as it is pulled in as a dependency during installation of the GNOME DE. However, it should be included here (for Intel support) if user decides not to install GNOME

So basically, the code would end up looking like this:
```shell
sudo pacman -S nvidia mesa
```

Reply:
> The lib32-nvidia-utils is effectively not required except for 32-bit (but will be pulled in by e.g. steam as a dependency).
oh ok

> lib32-virtualgl is actually redundant
yeah i forgot to remove it
> I hadn't included the mesa package here, as it is pulled in as a dependency during installation of the GNOME DE. However, it should be included here (for Intel support) if user decides not to install GNOME
i almost never install gnome i install xfce DE or WM
and mesa is needed as well for nvidia